proxyd/consensus_poller.go

```diff
@@ -318,18 +336,22 @@ func (cp *ConsensusPoller) UpdateBackend(ctx context.Context, be *Backend) {
 	}
 
 	latestBlockNumber, latestBlockHash, err := cp.fetchBlock(ctx, be, "latest")
-	if err != nil {
-		log.Warn("error updating backend - latest block", "name", be.Name, "err", err)
+	if err != nil || latestBlockNumber == 0 {
+		log.Warn("error updating backend - latest block will not be updated", "name", be.Name, "err", err)
+		latestBlockNumber = bs.latestBlockNumber
+		latestBlockHash = bs.latestBlockHash
 	}
 
 	safeBlockNumber, _, err := cp.fetchBlock(ctx, be, "safe")
```

Comment:
safe could potentially be zero as well

Reply:
Agreed I can update this so that if its zero too we return